src/js/components/put-away/PutAwayPage.jsx

```diff
@@ -342,6 +342,7 @@ class PutAwayPage extends Component {
         }
         <button
           type="button"
+          disabled={this.state.selection.size > 0 ? null : true}
```

Comment:
it should be 'false' instead of 'null' value in case of this.state.selection.size > 0